iFixit/react-commerce
# Strapi: Allow scheduling content to be published / unpublished

I decided to go with https://github.com/ComfortablyCoding/strapi-plugin-publisher as it is slightly more robust: More devs, longer lived, more releases, newer releases, more downloads etc

QA
---

* I had to run the following to kick over strapi locally
	* `cd backend/ &&  rm -rf .cache/ node_modules/ dist/`
	* Then `cd ../ && pnpm install:all`
* After running `pnpm dev` strapi should allow scheduling
* Note: The scheduling UI does not appear until after the first save of an item

Before:
![image](https://github.com/iFixit/react-commerce/assets/857362/06d17058-d280-44ed-aace-3088b85727b4)

Click save once, then you should see: 
![image](https://github.com/iFixit/react-commerce/assets/857362/5243b481-7c26-4fcf-8c07-0490286435d0)


* Note: [You can not see if something is scheduled in the list view](https://github.com/ComfortablyCoding/strapi-plugin-publisher/issues/95)

Closes: https://github.com/iFixit/ifixit/issues/50009
